src/KubernetesClient/KubernetesRequest.cs

```diff
@@ -0,0 +1,769 @@
+using System;
```

Comment:
This file needs formatting. in particular, space between `if` and `(...)`

Reply:
I assume that applies to `for` and `while` as well. I'll go reformat the whole thing if people think the change is worth having. :-)  Just let me know what changes are needed.